### DISCUSSION TITLE
Differential drive platform support in YARP

### DISCUSSION THREAD

**Discussion by pauljurczak** [robotology/community#59] [Q&A] [answered]
I know that YARP is specialized for humanoid robots. Is there off-the-shelf support for any differential drive robotic platform?

**Comment by pattacini** [comment, marked as answer]
Hi @pauljurczak 

We have used YARP for developing the control of an holonomic mobile platform called [**`iKart`**](http://wiki.icub.org/wiki/IKart). The relative software is available at https://github.com/robotology/ikart.

Compared with _iKart_, a standard differential drive robot turns to be much simpler to control, even though to my knowledge I don't know whether there might be already some YARP code for that.

Interestingly, quickly browsing the net, I've found this [link](http://www.openrobots.org/morse/doc/1.3/user/actuators/v_omega_diff_drive.html) out of a library called [**`MORSE`**](http://www.openrobots.org/morse/doc/stable/morse.html) which apparently provides a simulated differential drive control also for YARP.
